recipe/conda_build_config.yaml

```diff
@@ -6,7 +6,10 @@ cudnn:                                          # [linux64]
 - undefined                                     # [linux64]
 cdt_name:                                       # [linux64]
 - cos7                                          # [linux64]
```

Comment:
By the way, all those lines (1-8) should be removed as well.

Reply:
done.